src/Generators/ServiceGenerator.php

```diff
@@ -4,14 +4,15 @@
 
 use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
+use RonasIT\Support\DTO\RelationsDTO;
 use RonasIT\Support\Exceptions\ClassNotExistsException;
 use RonasIT\Support\Events\SuccessCreateMessage;
 
 class ServiceGenerator extends EntityGenerator
 {
-    public function setRelations($relations)
+    public function setRelations(RelationsDTO $relations)
```

Comment:
seems we may remove this logic at all as the parent method make the same but put these fields to the `integer_required` category which is the same for this generator. **Please check it before removing**